src/pages/404.tsx

```diff
@@ -1,12 +1,28 @@
-import * as React from "react"
-import type { PageProps } from "gatsby"
-import Layout from "@lekoarts/gatsby-theme-minimal-blog/src/components/layout"
+import * as React from "react";
+import { PageProps, graphql } from "gatsby";
```

Comment:
@kaleemullahdev  404 page template looks broken on the defaultLanguage (en) 
Please check http://localhost:8000/dasdas

![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/cd91ee44-2471-4e3c-a1d3-0b8e0e251c79)

Looks ok for secondary language.
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/8aba3cd0-2813-4510-acc1-0daacc367bdb)